types/leaflet/index.d.ts

```diff
@@ -550,12 +550,16 @@ export namespace tileLayer {
     function wms(baseUrl: string, options?: WMSOptions): TileLayer.WMS;
 }
 
+export type CrossOrigin = boolean | string;
```

Comment:
any reason to `export` the type?

Reply:
I was under the impression that all types should to be exported even if it seems unnecessary based on this documentation: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md. If I run `npm run lint leaflet` after making the change to this:
```ts
type CrossOrigin = boolean | string;
```

I get the following output:
```
$ npm run lint leaflet

> definitely-typed@0.0.3 lint /my/path/Documents/dev/DefinitelyTyped
> dtslint types "leaflet"

Error: /my/path/Documents/dev/DefinitelyTyped/types/leaflet/index.d.ts:552:6
ERROR: 552:6  strict-export-declare-modifiers  All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md

    at /my/path/Documents/dev/DefinitelyTyped/node_modules/dtslint/bin/index.js:160:19
    at Generator.next (<anonymous>)
    at fulfilled (/my/path/Documents/dev/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
    at <anonymous>
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! definitely-typed@0.0.3 lint: `dtslint types "leaflet"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the definitely-typed@0.0.3 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /my/path/.npm/_logs/2019-02-13T01_15_08_050Z-debug.log
```

Based on the error I think I would have to turn automatic exporting off for the whole module, which seems like it should be discussed in a separate issue.

Reply:
My bad, didn’t catch that requirement!